blocks/v2-stories-carousel/v2-stories-carousel.js

```diff
@@ -0,0 +1,160 @@
+import {
+  ffetch,
+} from '../../scripts/lib-ffetch.js';
+import {
+  createOptimizedPicture,
+  getOrigin,
+} from '../../scripts/lib-franklin.js';
+import { createElement } from '../../scripts/scripts.js';
+
+const listenScroll = (carousel) => {
+  const elements = carousel.querySelectorAll('.v2-stories-carousel-items > *');
+  const io = new IntersectionObserver((entries) => {
+    entries.forEach((entry) => {
+      if (entry.isIntersecting) {
```

Comment:
I tested the carousel on a desktop (with a high-resolution 2500px+) and I found a weird behavior. I suspect that's because more than one slide can be active at the same time.
Steps to reproduce:
1. Load the page on the 2500px + device
2. Click the right button -> Slide switch to the second item
3. Click the right button -> not working - nothing happened

Reply:
BTW the magic viewport is `2141px`.

Reply:
@cogniSyb @Lakshmishri  what should we do here? Should we use % instead of fixed size, so we make sure that this problem does not appear in huge resolutions?

Reply:
It is not fixed. Did we decide that we won't fix it?